src/video/SDL_egl.c

```diff
@@ -1202,18 +1202,19 @@ int SDL_EGL_SwapBuffers(_THIS, EGLSurface egl_surface)
     return 0;
 }
 
-void SDL_EGL_DeleteContext(_THIS, SDL_GLContext context)
+int SDL_EGL_DeleteContext(_THIS, SDL_GLContext context)
 {
     EGLContext egl_context = (EGLContext)context;
 
     /* Clean up GLES and EGL */
     if (!_this->egl_data) {
-        return;
+        return -1;
```

Comment:
What's the error here?

Reply:
I think this is the PSP backend that may return an error. so the sys psp gl_delete needs to returns an int. so all other backends need to returns an int.

(and so no egl_data sets, seems an here ?!)

Reply:
In that case, these should probably return 0 (success, nothing to do)

Reply:
fixed